Create documentation for filesystem repositories (#4287)
alcionai/corso: CHANGELOG.md

## [Unreleased] (beta)

### Added

> - Users can now create corso repositories on local or network attached storage.

Reviewer: ```suggestion
- Enables local or network-attached storage for Corso repositories.
```

Author: fancy
